__main__.py

```diff
@@ -108,76 +120,133 @@ def args_to_list(platform, projects):
     return projects_list
 
 
+# === COMMAND HANDLERS ===
+
+def handle_bitrise_builds(args):
+    client = BitriseClient()
+    client.bitrise_builds_detailed_info()
+
+
+def handle_bugzilla_desktop_bugs(args):
+    client = BugzillaClient()
+    client.bugzilla_query_desktop_bugs()
+
+
+def handle_bugzilla_meta_bugs(args):
+    client = BugzillaClient()
+    client.bugzilla_meta_bug(meta_bug_id=args.meta_bug_id)
+
+
+def handle_bugzilla_qe_verify(args):
+    client = BugzillaClient()
+    client.bugzilla_qe_verify()
+
+
+def handle_confluence_build_validation(args):
+    api_confluence.page_report_build_validation()
+
+
+def handle_confluence_updates(args):
+    api_confluence.main()
+
+
+def handle_github_issue_regression(args):
+    client = GithubClient()
+    client.github_issue_regression(args.project)
+
+
+def handle_jira_qa_requests(args):
+    client = JiraClient()
+    client.jira_qa_requests()
+
+
+def handle_jira_qa_needed(args):
+    client = JiraClient()
+    client.jira_qa_needed()
```

Comment:
you would need to add here `client.jira_qa_requests_new_issue_types()`

https://github.com/mozilla-mobile/testops-dashboard/pull/157/files#diff-5b63cf71490a87d1d0956d82c6eb688a4c95c7b0633c145c2164865eb4c0a8ceL155

Reply:
I don't see that query being executed in the check :( it is running in other checks but not here.
Looking at this run:
https://github.com/mozilla-mobile/testops-dashboard/actions/runs/16769918767/job/47482511074

versus one in main where it is: https://github.com/mozilla-mobile/testops-dashboard/actions/runs/16751080250/job/47421208021

in the Jira-qa-request section, you should see two queries in the logs (you can filter by Total issues retrieved, there should be two, one for each query)